region/maridia/inner-yellow/Watering Hole.json

```diff
@@ -142,6 +142,57 @@
         "Requires a pixel-perfect freeze in order to run onto and off of the frozen Zebs."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit) - Frozen Zeb",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "h_frozenEnemyRunway",
+        {"canShineCharge": {"usedTiles": 13, "openEnd": 0}},
+        {"refill": ["Energy"]},
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
+        "canRModeSparkInterrupt",
+        {"or": [
+          "Gravity",
+          {"disableEquipment": "Ice"}
+        ]}
+      ],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Freeze the left Zeb at a pixel perfect position to extend a runway across the top. With the shinecharge,",
+        "drop into the left chasm then quickly jump under the Choot to get hit by it and interrupt.",
+        "Freeze the Choot to help get back out, if needed."
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit) - Double Frozen Zeb Runway",
```

Comment:
You can remove this and combine it with the other strat

Reply:
This might be another thing that should be addressed on MapRando side, but I have concerns about 'hiding" notable strats (such as the Double Frozen Zeb Runway) when they get packaged into another strat like R-Mode Spark.

E.g. if this were displayed on MapRando it might look like this:
> Caterpillar Room
>   Elevator
>   Upper Left Door: G-Mode Setup, Get Hit By Zero
> Watering Hole
>   Right Door: R-Mode Spark Interrupt - Double Frozen Zeb Runway

If the strat was combined with the single zeb, it would not be known that the notable strat was being called for to use a longer runway.

That being said, if there's lenience in the double-Zeb freeze then the notable can be taken out as a factor.

Reply:
I think it's ok, as it will still ensure that the player toggled that notable on ... there can definitely be exceptions, but i think its fine here